scripts/deb-systemd.sh

```diff
@@ -32,7 +32,7 @@ After=network.target
 [Service]
 User=ubuntu
 WorkingDirectory=/home/ubuntu
-ExecStart=/usr/local/bin/jinad
+ExecStart=/usr/local/jina/bin/jinad
```

Comment:
We need to start `jinad` by pasing the right workspace.
Please change this to (verify before merging)
```bash
ExecStart=/usr/local/jina/bin/jinad --workspace /usr/local/jina/tmp
```

Reply:
thx

Reply:
In this PR, we change pip prefix from `/usr/local` to `/usr/local/jina`, which  install binary of jinad in `/usr/local/jina/bin`

Reply:
My comment is only for passing the custom workspace directory while starting JinaD.

Reply:
OK, I get it.